src/features/vars-schema/env-vars.ts

```diff
@@ -267,6 +267,12 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
 					'Define the file name of the DTB to be used. Only supported by supervisor versions >= v11.14.2.',
 				will_reboot: true,
 			},
+			BALENA_HOST_EXTLINUX_isolcpus: {
+				type: 'string',
+				description:
+					'Allows to isolate CPU cores from kernel scheduler. See kernel parameters. Only supported by supervisor versions >= v7.10.0.',
```

Comment:
Would be nice to rephrase the `See kernel parameters` to `...using/by specifying kernel parameters`.
Bonus points if you manage to get an example as well. Afaict `1,2,3` might be an example, but plz confirm that.

Reply:
Very less information, I found all this and pushed the example. 
- Open this link using Chrome --> https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/7/html/tuning_guide/isolating_cpus_using_tuned-profiles-realtime#:~:text=swap%20console%3DttyS0%2C115200n81-,isolcpus%3D0%2C4,-Isolating%20CPUs%20Using

- https://rt-labs.com/docs/p-net/linuxtiming.html#run-the-application-on-a-separate-processor-core

- https://www.suse.com/support/kb/doc/?id=000017747

Reply:
okay, this format of the example is confirmed. Check FD conversation: https://www.flowdock.com/app/rulemotion/r-supervisor/threads/g5SxblwceDmrBTT8ROiR3BQCyIm